File: Iteration2/code/CSE3063GRP12/src/features/notification/NotificationView.java
package features.notification;

import java.util.ArrayList;

import core.models.concretes.Notification;


public class NotificationView {

    public void showNotificationList(ArrayList<Notification> notificationList){
        for (int i = notificationList.size()-1; i >= 0; i--) {
            Notification notification = notificationList.get(i);
            if(notification.isRead() == false){
                System.out.println("\033[1m" + notification.getMessage() + "\033[0m "); // Bold text using escape codes
            }
            else{
                System.out.println(notification.getMessage());
            }
        }
    }    
    public void showQuitMessage(){
        System.out.print("Press q to return Main Menu: ");
    }
}

File: Iteration3/code/CSE3063GRP12/src/features/notification/NotificationView.py
class NotificationView:

    def show_notification_list(self, notification_list):
            # Iterating through the list in reverse order as in the original Java code
            for notification in reversed(notification_list):
                # Check if the notification is unread
                if not notification.get("is_read"):
                    # Printing in bold in Python is a bit different than in Java
                    print(f"\033[1m{notification.get('message')}\033[0m")  # Bold text
                else:
                    print(notification.get('message'))